### PR TITLE
feat(motor-control): adjustable motor current through CAN

### PR DESCRIPTION
**Pull request by ahiuchingau**
This PR allows us to change the TMC2130 driver motor currents via CAN in mA (in the form of a `uint32_t`)!

- Added new CAN message & ID for `WriteMotorCurrentRequest` that takes both the hold current (standstill) and the run current
- Updated motor driver task to handle the new can message
- Reconfigured the TMC2130 class so each subsystem can customize the driver's `R sense` and `Vsf` values that are crucial to motor current calculations

### CODE REVIEW

File: include/motor-control/core/tasks/motor_driver_task.hpp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <cmath>
```
Comment (sfoster1) [inline include/motor-control/core/tasks/motor_driver_task.hpp:3]: do we need these includes?

File: include/motor-control/core/tmc2130.hpp [hunk 2]
```diff
@@ -126,6 +132,21 @@ class TMC2130 {
         return true;
     }
 
+    [[nodiscard]] auto convert_to_tmc2130_current_value(uint32_t c) const
+        -> uint32_t {
+        constexpr const float SMALL_R = 0.02;
+        const auto SQRT_TWO = static_cast<float>(std::sqrt(2));
```
Comment (sfoster1) [inline include/motor-control/core/tmc2130.hpp:138]:
```suggestion
        constexpr auto SQRT_TWO = static_cast<float>(std::sqrt(2));
```

**Review by sfoster1 — APPROVED** [review]
awesome! maybe an include or two that isn't needed but this is great

File: include/motor-control/core/tmc2130.hpp [hunk 3]
```diff
@@ -126,6 +132,21 @@ class TMC2130 {
...
+        auto FLOAT_CONSTANT = static_cast<float>(
+            SQRT_TWO * 32.0 * (_current_config.r_sense + SMALL_R) /
+            _current_config.v_sf);
+        auto fixed_point_constant = static_cast<uint32_t>(
+            FLOAT_CONSTANT * static_cast<float>(1LL << 16));
```
Comment (Laura-Danielle) [inline include/motor-control/core/tmc2130.hpp:143]: we have functions that allow you to multiply and divide fixed point

Comment (ahiuchingau) [reply include/motor-control/core/tmc2130.hpp:146]: Will add this unsigned fixed point multiply func to that file

**Pull request merged**